### ISSUE TITLE
Add blackhole Attribute to aws_ec2_transit_gateway_route

### ISSUE THREAD

**Issue by stockhausenj** [terraform-providers/terraform-provider-aws#9046]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

With the AWS API it's possible to declare a transit gateway route table route as a blackhole. Which means that traffic matching the route will be dropped. 
API: https://docs.aws.amazon.com/cli/latest/reference/ec2/create-transit-gateway-route.html
If a route is a blackhole then the attribute transit_gateway_attachment_id is ignored.

### New or Affected Resource(s)

* aws_ec2_transit_gateway_route

### Potential Terraform Configuration

```hcl
resource "aws_ec2_transit_gateway_route" "example" {
  destination_cidr_block              = "10.0.0.0/8"
  blackhole                                   = true
  transit_gateway_route_table_id = "${aws_ec2_transit_gateway.example.association_default_route_table_id}"
}
```

### References

* #0000

**Comment by kellersyf** [comment]
I'm really surprised this isn't here already.  Implementation should just be adding the `blackhole` flag and not setting the `transit-gateway-attachment-id` field.

Here's the [AWS API reference](https://docs.aws.amazon.com/cli/latest/reference/ec2/create-transit-gateway-route.html) for the feature.

I'm happy to provide any other information, although I unfortunately do not have any experience in Go. =\

**Comment by bflad** [comment]
A new `blackhole` argument for the `aws_ec2_transit_gateway_route` resource has been merged and will release with version 2.18.0 of the Terraform AWS Provider, likely in the next day. Thanks, @anouvel!

**Comment by bflad** [comment]
This has been released in [version 2.18.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2180-july-05-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

For further feature requests or bug reports with this functionality, please create a [new GitHub issue](https://github.com/terraform-providers/terraform-provider-aws/issues/new/choose) following the template for triage. Thanks!
